app/workers/comments/bust_cache_worker.rb

```diff
@@ -0,0 +1,14 @@
+module Comments
+  class BustCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority
+
+    def perform(comment_id, service = "EdgeCache::Commentable::Bust")
```

Comment:
I think we can remove the `service` argument here and just call it in the body of the worker

Reply:
can do but looks like this is pattern is used (at least in a couple of other bust cache jobs) and allows some flexibility to inject another dependency if need be?

Reply:
Unfortunately we don't have a clear style guide and I understand the confusion. The thing is that we don't actually use that feature (injecting dependency when running a job) and we do only for testing, so we might as well get rid of it

Reply:
Gotcha, will clean it up